framework/resources/Functional/table_stats/stats/positive/meta_stats.sql

```diff
@@ -1 +1,2 @@
-explain plan including all attributes for select l_orderkey,l_partkey,l_shipdate,l_shipmode from `table_stats/lineitem_parquet_partitioned_hive_date` where dir0 = 'dt=2014-10-22' and l_shipmode='AIR';
+set `planner.statistics.use` = true;
```

Comment:
if there are even no of queries you have to add a --@test to the query u want the result on.
My suggestion would be to reset `planner.statistics.use` when u set them

Reply:
Yes, I would recommend first doing a SET and then a RESET. Here there is actually just one query to be tested.

Reply:
Did a reset.